### PR TITLE
Minimal example and test

### PR DESCRIPTION
**Pull request by sjmiller609**
rebase on main after merging this PR https://github.com/tembo-io/tembo/pull/441

This PR checks this tembo.toml files works in CI:

```
[minimal]
environment = "prod"
instance_name = "minimal"
```

### CODE REVIEW

File: tembo-cli/tests/tomls/minimal/tembo.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[minimal]
+environment = "prod"
+instance_name = "minimal"
```
Comment (sjmiller609) [inline tembo-cli/tests/tomls/minimal/tembo.toml:3]: In this PR, I'm seeking to get this toml to work and add docker test of it to CI

File: .github/workflows/cargo-test.yaml [hunk 2]
```diff
@@ -3,8 +3,7 @@ name: Cargo lint and test
 on:
   push:
     branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+  pull_request: {}
```
Comment (sjmiller609) [inline .github/workflows/cargo-test.yaml:6]:
```suggestion
  pull_request:
    branches: [ "main" ]
```

File: tembo-cli/src/cli/file_utils.rs [hunk 3]
```diff
@@ -33,6 +33,13 @@ impl FileUtils {
             info!("Tembo {} file exists", file_name);
             return Ok(());
         }
+
+        // Create all missing directories in the path
```
Comment (shahadarsh) [inline tembo-cli/src/cli/file_utils.rs:37]: Nice!

File: tembo-cli/tests/tomls/minimal/tembo.toml (hunk 1 above)
Comment (shahadarsh) [reply tembo-cli/tests/tomls/minimal/tembo.toml:3]: This is awesome! Thank you.

**Review by shahadarsh — APPROVED** [review]
LGTM :thumbsup:

**Pull request merged**